epios/tests/test_post_process.py

```diff
@@ -0,0 +1,559 @@
+                                        '0.0.0.0': ['InfectASympt'] * 6,
+                                        '0.0.0.1': ['InfectASympt'] * 6,
+                                        '0.0.0.2': ['InfectASympt'] * 6,
+                                        '1.0.0.0': ['InfectASympt'] * 6,
+                                        '1.0.0.1': ['InfectASympt'] * 6,
+                                        '1.0.0.2': ['InfectASympt'] * 6})
+        self.processor_non_responder2 = PostProcess(self.demo_data3, self.time_data3)
+
+    def test__init__(self):
+        try:
+            assert_frame_equal(self.processor.demo_data, pd.DataFrame({'ID': ['0.0.0.0', '0.0.0.1', '0.0.1.0',
+                                                                              '0.1.0.0', '0.2.0.0', '1.0.0.0'],
+                                                                       'age': [1, 81, 45, 33, 20, 60]}))
+            assert_frame_equal(self.processor.time_data, pd.DataFrame({'time': [0, 1, 2, 3, 4, 5],
+                                                                       '0.0.0.0': ['InfectASympt'] * 6,
+                                                                       '0.0.0.1': [0, 0, 0, 'InfectASympt',
+                                                                                   'InfectASympt', 'InfectASympt'],
+                                                                       '0.0.1.0': [0, 0, 'InfectASympt',
+                                                                                   'InfectASympt', 'InfectASympt',
+                                                                                   'InfectASympt'],
+                                                                       '0.1.0.0': [0, 0, 'InfectASympt',
+                                                                                   'InfectASympt',
+                                                                                   'InfectASympt', 'InfectASympt'],
+                                                                       '0.2.0.0': [0, 'InfectASympt', 'InfectASympt',
+                                                                                   'InfectASympt', 'InfectASympt',
+                                                                                   'InfectASympt'],
+                                                                       '1.0.0.0': [0, 0, 0, 0, 0, 'InfectASympt']}))
+        except AssertionError:
+            self.fail('init function error')
+
+    def test_sampled_result_s(self):
+        kwargs = {
+            'sample_strategy': 'Same',
+            'gen_plot': True,
+            'saving_path_sampling': self.path + 'sample.png'
+        }
+        res = self.processor('AgeRegion', 6, [0, 1, 2, 3, 4, 5], comparison=False, data_store_path=self.path, **kwargs)
+        self.assertAlmostEqual(res, [[0, 1, 2, 3, 4, 5], [1 / 6, 2 / 6, 4 / 6, 5 / 6, 5 / 6, 6 / 6]])
```

Comment:
Could generate these lists using range functions and list comprehensions, and then save them as variables so you don't have to write them out every time they are called in a test